0.2-Elliptic-Curve-Math.ipynb

```diff
@@ -364,7 +344,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.6.5"
```

Comment:
That's for the sake of consistency, all the rest of the notebooks share this version.

Reply:
Not sure how this one got the wrong version, but thanks for finding it!